### PR TITLE
ASP inverted index project

### PR DESCRIPTION
**Pull request by mojo-izadi**

### CODE REVIEW

File: phase9/Controllers/InvertedIndexController.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+using SampleLibrary;
+
+namespace ASP.Controllers;
+
+[ApiController]
+[Route("[controller]/[Action]")]
+public class InvertedIndexController : ControllerBase
+{
+    private InvertedIndex _invertedIndex = new ();
+    private FileReader _fileReader = new(@"Resources");
```
Comment (mojo-izadi) [inline phase9/Controllers/InvertedIndexController.cs]: we will handle this with dependency injection in our next push

**Review by M79Montazer — APPROVED** [review]
resources should not be included in PRs. other than that, good job :)

File: phase9/Controllers/InvertedIndexController.cs (hunk 1 above)
Comment (mojo-izadi) [reply phase9/Controllers/InvertedIndexController.cs]: We were going to handle it with dependency injection but unfortunately our library does not implement interfaces with both these classes

**Review by amirarab888 — APPROVED** [review]

File: phase9/Controllers/InvertedIndexController.cs [hunk 2]
```diff
@@ -0,0 +1,30 @@
+using Microsoft.AspNetCore.Mvc;
+using SampleLibrary;
+
+namespace ASP.Controllers;
+
+[ApiController]
+[Route("[controller]/hoho")]
```
Comment (RAAvenger) [inline phase9/Controllers/InvertedIndexController.cs]: hoho?!

**Review by RAAvenger — APPROVED** [review]
سلام
خسته نباشید

**Pull request merged**